src/cljs/rems/table.cljs

```diff
@@ -152,8 +152,8 @@
    (->> rows
         (sort-by #(get-in % [(:sort-column sorting) :sort-value])
```

Comment:
[nit] you could put str/lower-case here in the keyfn, instead of below in the comparison

Reply:
Done.